src/components/App/App.js

```diff
@@ -10,14 +10,15 @@ import LoadingSimple from '../UI/LoadingSimple/LoadingSimple'
 
 // Routes and Layout for public/unauthorized user
 import Layout from '../Layouts/PublicLayout'
+import SignIn from '../../pages/sign-in/SignIn'
 
 const App = ({ title }) => {
   // Get current location. if it's homepage =>  rendering landing
   const location = useLocation()
 
   let layoutToDisplay = ''
 
-  if (location.pathname === '/') {
+  if (location.pathname === '/' || location.pathname === '/sign-in') {
```

Comment:
maybe you should try to add sign-in component as a modal? 
it could be a modal, that will be displayed on top of Landing page, without any blurry efects for the background.
Needs to create a  Modal component, and to make is flexible for all of projects needs.